.azuredevops/pipelineTemplates/jobs.validateModulePester.yml

```
@@ -149,6 +149,12 @@ jobs:
               Run        = @{
                 Container = New-PesterContainer -Path (Join-Path '$(moduleRepoRoot)' 'arm' '.global' 'global.module.tests.ps1') -Data @{
                   moduleFolderPaths = $moduleFolderPaths
+                  tokensToAvoid = @{
```

tokensToAvoid sounds rude. Should we find a nicer name?

---

Isn't that the idea? :D Open for suggestions though

---

load it from a json file with "blacklist token names" :D

---

You gave me the inspiration @MariusStorhaug 
```suggestion
                  notAllowedHardcodedTokens = @{
```

---

`enforcedTokenList`?

---

Fair enough

---

Updated